ipatests/test_integration/test_installation.py

```diff
@@ -703,6 +703,30 @@ def install(cls, mh):
     def test_install_master(self):
         tasks.install_master(self.master, setup_dns=False)
 
+    def test_smoke_test_for_debug_mode(self):
+        """Test if an IPA server works in debug mode.
+        Related: https://pagure.io/freeipa/issue/8891
+        """
+
+        target_fname = paths.IPA_SERVER_CONF
+        assert not self.master.transport.file_exists(target_fname)
+
+        # set the IPA server in debug mode
+        server_conf = "[global]\ndebug=True"
+        self.master.put_file_contents(target_fname, server_conf)
+        assert self.master.transport.file_exists(target_fname)
+        self.master.run_command(["systemctl", "restart", "httpd"])
```

Comment:
Could you please replace the hardcoded service name with `knownservices.httpd.systemd_name`?
in the assumption that tests controller's OS is the same as the worker's one or better to query it from worker's ipaplatform.

Reply:
For example,
```python3
    result = host.run_command(                                                  
        [                                                                       
            "python3",                                                          
            "-c",                                                               
            (                                                                   
                "from ipaplatform.services import knownservices; "              
                "print(knownservices.named.systemd_name)"                       
            ),                                                                  
        ]                                                                       
    )                                                                           
    service_name = result.stdout_text.strip()                                   
    host.run_command(["systemctl", "restart", service_name])
```

Reply:
Sorry, no. I'll restart ipa instead.

Reply:
It's ok :)

Reply:
I've switched to restarting ipa.